qa/workunits/fs/misc/multiple_rsync.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/sh -ex
 
-sudo rsync -av --exclude local/ /usr/ usr.1
-sudo rsync -av --exclude local/ /usr/ usr.2
+
+# Populate with some arbitrary files from the local system.  Take
+# a copy to protect against false fails from system updates during test.
+export PAYLOAD=/tmp/multiple_rsync_payload.$$
+cp -r /usr/lib/ $PAYLOAD
```

Comment:
Oh, sounds awesome then! I should learn shell.
On Tue, Jun 2, 2015 at 8:57 PM Sage Weil <notifications@github.com> wrote:

> In qa/workunits/fs/misc/multiple_rsync.sh
> <https://github.com/ceph/ceph/pull/4831#discussion_r31592218>:
>
> > @@ -1,16 +1,26 @@
> >  #!/bin/sh -ex
> >
> > -sudo rsync -av --exclude local/ /usr/ usr.1
> > -sudo rsync -av --exclude local/ /usr/ usr.2
> > +
> > +# Populate with some arbitrary files from the local system.  Take
> > +# a copy to protect against false fails from system updates during test.
> > +export PAYLOAD=/tmp/multiple_rsync_payload.$$
> > +cp -r /usr/lib/ $PAYLOAD
>
> The $$ makes it unique.. are you worried about filling /tmp?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/4831/files#r31592218>.
>